ios/Exponent/Versioned/Core/Api/Cognito/RNAWSCognito.h

```diff
@@ -16,8 +16,8 @@
 #import <React/RCTUtils.h>
 #endif
 
-#import <JKBigInteger.h>
+#import "JKBigInteger.h"
```

Comment:
I think this might affect versioning: https://github.com/expo/expo/commit/0356aa6bb3f791e5fee71ebfc4e9509f4a2e550d#diff-c3c8410a579bdf88d36dc16ae9e63765aad4d79eb49802a56a596bcbb596fce2

It looks like the CI workflow for versioning wasn't triggered, so could you try running it locally and see if it passes?

Reply:
ios versioning breaks for a while so i cannot test the versioning. however i tried to backport the following changes to previous sdk code, versioned expo go still builds success. it might not be a big problem. maybe from [this commit](https://github.com/expo/expo/commit/90d5273bd1bf78a4532bc301d50522b02e3be41f) where based on cocoapods, there were something different.

let me know if you prefer to apply the following changes to previous sdk code as well.

```diff
diff --git a/ios/versioned-react-native/ABI40_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI40_0_0RNAWSCognito.h b/ios/versioned-react-native/ABI40_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI40_0_0RNAWSCognito.h
index 1bdd8b04df..6bbecd2955 100644
--- a/ios/versioned-react-native/ABI40_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI40_0_0RNAWSCognito.h
+++ b/ios/versioned-react-native/ABI40_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI40_0_0RNAWSCognito.h
@@ -16,7 +16,7 @@
 #import <ABI40_0_0React/ABI40_0_0RCTUtils.h>
 #endif

-#import <JKBigInteger.h>
+#import "JKBigInteger.h"

 @interface ABI40_0_0RNAWSCognito : NSObject <ABI40_0_0RCTBridgeModule>

diff --git a/ios/versioned-react-native/ABI41_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI41_0_0RNAWSCognito.h b/ios/versioned-react-native/ABI41_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI41_0_0RNAWSCognito.h
index d9244e0056..23af830278 100644
--- a/ios/versioned-react-native/ABI41_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI41_0_0RNAWSCognito.h
+++ b/ios/versioned-react-native/ABI41_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI41_0_0RNAWSCognito.h
@@ -16,7 +16,7 @@
 #import <ABI41_0_0React/ABI41_0_0RCTUtils.h>
 #endif

-#import <JKBigInteger.h>
+#import "JKBigInteger.h"

 @interface ABI41_0_0RNAWSCognito : NSObject <ABI41_0_0RCTBridgeModule>

diff --git a/ios/versioned-react-native/ABI42_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI42_0_0RNAWSCognito.h b/ios/versioned-react-native/ABI42_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI42_0_0RNAWSCognito.h
index c44160b802..5985d01f52 100644
--- a/ios/versioned-react-native/ABI42_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI42_0_0RNAWSCognito.h
+++ b/ios/versioned-react-native/ABI42_0_0/Expo/ExpoKit/Core/Api/Cognito/ABI42_0_0RNAWSCognito.h
@@ -16,7 +16,7 @@
 #import <ABI42_0_0React/ABI42_0_0RCTUtils.h>
 #endif

-#import <JKBigInteger.h>
+#import "JKBigInteger.h"

 @interface ABI42_0_0RNAWSCognito : NSObject <ABI42_0_0RCTBridgeModule>

```

Reply:
> however i tried to backport the following changes to previous sdk code, versioned expo go still builds success

okay sounds good then 👍  we can keep this in mind during iOS versioning in case there are issues then

Reply:
I think backporting the change to previous versions makes sense, just to keep things consistent

Reply:
thanks for the catch up. updated the pr accordingly, please take a look when you getting a chance.